webapp/src/client/app/modules/web-client/ard/web-client-ard.component.html

```diff
@@ -67,6 +67,6 @@
     </div>
   </div>
 
-  <iron-remote-gui-vnc #ironGuiElementVnc debugwasm="INFO" verbose="true" scale="fit" flexcenter="true"></iron-remote-gui-vnc>
+  <iron-remote-gui-vnc #ironGuiElementVnc debugwasm="OFF" verbose="false" scale="fit" flexcenter="true"></iron-remote-gui-vnc>
```

Comment:
Parameters is not already OFF and false by default ?
If it's the case you can just remove them from the declaration.